packages/hash/frontend/src/components/hooks/blockProtocolFunctions/ontology/useBlockProtocolAggregateDataTypes.ts

```diff
@@ -37,7 +37,11 @@ export const useBlockProtocolAggregateDataTypes = (): {
        *   or doing it from here.
        *   https://app.asana.com/0/1202805690238892/1202890614880643/f
        */
-      const response = await aggregateFn({});
+      const response = await aggregateFn({
+        variables: {
+          dataTypeResolveDepth: 255,
```

Comment:
What is the reason for this being 255 while the other ontology aggregate hooks default to `0` across all resolve depths?

Reply:
No good reason, and it should be consistent. I'm happy to change it to 0, but I think generally we need to figure out if the aggregate methods should return the full depths or not. I think I'd err on the side of changing the other aggregate methods to have 255

Reply:
That's been done https://github.com/hashintel/hash/pull/1220/commits/7163273163e39101eae6652971af6ae1e82f8353

Reply:
Had missed some https://github.com/hashintel/hash/pull/1220/commits/b1b81efcf907420c791b5197dec3c1e87dd86f6f